xyz.ayrsd.AudioStellar.yml

```diff
@@ -0,0 +1,319 @@
+
+  - name: glfw
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+    - "-DCMAKE_BUILD_TYPE=RelWithDebInfo"
+    - "-DBUILD_SHARED_LIBS:BOOL=ON"
+    sources:
+    - type: archive
+      url: https://github.com/glfw/glfw/releases/download/3.3.2/glfw-3.3.2.zip
+      sha256: 08a33a512f29d7dbf78eab39bd7858576adcc95228c9efe8e4bc5f0f3261efc7
+    cleanup:
+    - "/include"
+    - "/lib/pkgconfig"
+
+  - name: liburiparser
+    buildsystem: cmake-ninja
+    config-opts:
+    - "-DURIPARSER_BUILD_DOCS=OFF"
+    - "-DURIPARSER_BUILD_TESTS=OFF"
+    cleanup:
+    - "/bin/uriparse"
+    sources:
+    - type: archive
+      url: https://github.com/uriparser/uriparser/releases/download/uriparser-0.9.4/uriparser-0.9.4.tar.bz2
+      sha256: b7cdabe5611408fc2c3a10f8beecb881a0c7e93ff669c578cd9e3e6d64b8f87b
+
+  - name: pugixml
+    buildsystem: cmake-ninja
+    build-options:
+      cflags: "-fPIC"
+      cxxflags: "-fPIC"
+    sources:
+    - type: archive
+      url: https://github.com/zeux/pugixml/releases/download/v1.10/pugixml-1.10.tar.gz
+      sha256: 55f399fbb470942410d348584dc953bcaec926415d3462f471ef350f29b5870a
+
+  - name: poco
+    buildsystem: cmake-ninja
+    sources:
+    - type: archive
+      url: https://github.com/pocoproject/poco/archive/refs/tags/poco-1.11.8-release.tar.gz
+      sha256: a59727335a9bf428dc1289cd8ce84f9c1749c1472a0cd3ae86bec85be23d7cbe
+      
+  - name: libdaemon
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: http://git.0pointer.net/clone/libdaemon.git
+        commit: 9fcc28e0e8f84968d1fb8b6d544a42efb13803ec
+        disable-shallow-clone: true # dumb http transport does not support shallow capabilities
+        
+  - name: libevent
+    buildsystem: autotools
+    sources:
+      - type: git
+        url: https://github.com/libevent/libevent
+        tag: release-2.1.12-stable
+        
+  - name: avahi
+    buildsystem: autotools
+    config-opts:
+      - "--with-distro=none"
+      - "--disable-gtk3"
+      - "--disable-mono"
+      - "--disable-qt5"
+      - "--disable-manpages"
+      - "--disable-python"
+      - "--with-systemdsystemunitdir=no"
+    sources:
+      - type: git
+        url: https://github.com/lathiat/avahi
+        tag: v0.8
+  
+  - name: libsnappy
+    buildsystem: cmake-ninja
+    config-opts:
+      - "-DBUILD_SHARED_LIBS=ON"
+      - "-DSNAPPY_BUILD_TESTS=OFF"
+      - "-DCMAKE_BUILD_TYPE=Release"
+    sources:
+      - type: git
+        url: https://github.com/google/snappy
+        tag: 1.1.7
+
+  - name: openframeworks
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/lib/oF
+      - mv * ${FLATPAK_DEST}/lib/oF
+      - sh ${FLATPAK_DEST}/lib/oF/scripts/linux/compileOF.sh -j$FLATPAK_BUILDER_N_JOBS
+    sources:
+      - type: archive
+        url: https://github.com/openframeworks/openFrameworks/releases/download/0.12.0/of_v0.12.0_linux64gcc6_release.tar.gz
```

Comment:
so how does this work on aarch64?

Reply:
you mean ARM ? it doesn't. it just works for linux64 for now. 
i've run a few tests and could make it run on a raspi, but i guess I'll just add raspi support in the future

Reply:
linux 64 mean linux, 64-bits. which techincally aarch64 is. but I know most people make that x86_64 only.

you are building an aarch64 flatpak right now. and I know openFrameworks have been fixed for support more than x86_64. Also why not building from source instead?